Code/client/Games/Skyrim/Actor.cpp

```diff
@@ -352,6 +352,19 @@ void Actor::StopCombat() noexcept
     s_pStopCombat(this);
 }
 
+void Actor::RemoveSpell(MagicItem* apSpell) noexcept
```

Comment:
This function will cause the `HookRemoveSpell` hook to trigger. Instead, the hook should be moved to `Actor.cpp`, and this function should call `RealRemoveSpell` instead of fetching the address of `RemoveSpell`. For an example on how to do this, check out `InitiateMountPackage`, `RealInitiateMountPackage` and `HookInitiateMountPackage`.